src/flamenco/vm/fd_vm_interp.c

```diff
@@ -68,6 +70,8 @@ fd_vm_exec_trace( fd_vm_t * vm ) {
 
   fd_vm_shadow_t * FD_RESTRICT shadow = vm->shadow;
 
+  uchar is_multi_region = 0;
```

Comment:
Could we not make this a global variable, but instead scope it to each instruction? Since it's only set-and-read inside a single instruction at a time. Having it here is confusing, because it makes it seem like it's part of the vm's state, whereas it's actually part of the instructions state. Minimising the scope of variables makes code easier to understand and improves performance.

Reply:
sure

Reply:
thank-you!